src/main/java/com/starbucks/controllers/RestController.java

```diff
@@ -1,26 +1,38 @@
 package com.starbucks.controllers;
 
 import com.starbucks.model.Order;
-import com.starbucks.repositary.OrderRepositary;
-import org.springframework.web.bind.annotation.PostMapping;
-import org.springframework.web.bind.annotation.RequestBody;
-import org.springframework.web.bind.annotation.RequestMapping;
+import com.starbucks.service.OrderService;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.UUID;
 
 @org.springframework.web.bind.annotation.RestController
-@RequestMapping("/order")
```

Comment:
You should put @RequestMapping("/orders") here instead of putting it above every controller method.